src/pages/public_registration/ActionPanel.tsx

```diff
@@ -19,23 +20,61 @@ export const ActionPanel = ({
   isLoadingData,
 }: ActionPanelProps) => {
   const { t } = useTranslation();
+  const [tabValue, setTabValue] = useState(0);
 
   return (
-    <Paper
-      elevation={0}
-      data-testid={dataTestId.registrationLandingPage.tasksPanel.panelRoot}
-      sx={{ gridArea: 'tasks' }}>
-      <StyledPaperHeader>
-        <Typography color="inherit" variant="h1">
-          {t('common.tasks')}
-        </Typography>
-      </StyledPaperHeader>
-      <ActionPanelContent
-        tickets={tickets}
-        refetchData={refetchRegistrationAndTickets}
-        isLoadingData={isLoadingData}
-        registration={registration}
-      />
+    <Paper elevation={0} data-testid={dataTestId.registrationLandingPage.tasksPanel.panelRoot}>
+      <Box sx={{ color: 'primary.contrastText' }}>
+        <Tabs
+          value={tabValue}
+          onChange={(_, newValue) => setTabValue(newValue)}
+          aria-label="action-panel-tabs"
+          sx={{ bgcolor: 'primary.main', px: '0.5rem' }}
```

Comment:
Bare min subjektive mening, men den lille gappen til høyre gjorde meg litt utilpass 😬 
```suggestion
          sx={{ bgcolor: 'primary.main' }}
```

Reply:
Ja, egentlig enig, men samtidig så synes jeg det tydeliggjør streken mot den hvite bakgrunnen på nettsiden. Antar det derfor er designet slik i skissene. Så tror jeg bare lar den være.

Reply:
Ja, så litt på det jeg og🤔 Prøvde å finne en måte å legge til en `boxShadow` (som i headeren), men fant ikke noe i farta.